.github/ISSUE_TEMPLATE/don-t-create-issues-here--go-to-https---github-com-jenkinsci-ghprb-plugin-issues.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Don't create issues here, go to https://github.com/jenkinsci/ghprb-plugin/issues
+about: No issues here, go to https://github.com/jenkinsci/ghprb-plugin/issues
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+### To report issues, go to the project location at https://github.com/jenkinsci/ghprb-plugin/issues
```

Comment:
What is this file?  It doesn't look like it provides anything beneficial to the project because

1. We're on GitHub.
2. GitHub issues are pretty standard across all GitHub projects unless it is disabled.
3. For projects that have GitHub issues disabled, they usually reference where issues are tracked off-site.